docs/concepts/storage-profiles-content.rst

```diff
@@ -0,0 +1,19 @@
+Flocker Storage Profiles are a way to help you choose different levels of service from your storage provider.
```

Comment:
Can we please reword this from an end-user application point of view? For example: 

``` Flocker Storage Profiles are a way to help your application's storage needs by choosing the best set of service attributes from your storage provider .```